tests/moduletool/conftest.py

```diff
@@ -147,6 +162,9 @@ def modules_repo(git_modules_dir) -> str:
     add_file(mod12, "file", "test", "release version 4.0.0.dev0", version="4.0.0.dev0")
     add_file(mod12, "file", "test", "release version 4.0.0", version="4.0.0")
 
+    mod13 = make_module_simple(reporoot, "mod13", version="1.2.3", depends=[("mod11", "<4.2.0")])
+    update_requires(mod13, [["mod11"], ["<4.1.2"]], "add dependency", version="1.2.4")
```

Comment:
I find the second argument hard to interpret, but maybe that also linked to the fact that `update_requires` doesn't have a docstring.